src/index.js

```diff
@@ -4,7 +4,10 @@ const morgan = require('morgan')
 const chalk = require('chalk')
 const mongoose = require('mongoose')
 const users = require('./routes/api/users')
+import errorHandler from './utils/errorHandler'
+
 import emailVerify from './routes/endpoints/users'
+import mentees from "./routes/mentees";
```

Comment:
Move the mentees folder inside the endpoints/api.
```suggestion
import mentees from "./routes/api/mentees";
```